### PR TITLE
16 Plug & Play / Handheld additions (4 working)

### PR DESCRIPTION
**Pull request by mamehaze**
new WORKING machines
---------------
Idaten Sho Gekisou - Idaten Jump (Japan) [TeamEurope, David Haywood]
Let's Build a Town: Construction Cars 5 (Japan) [TeamEurope, David Haywood]
ConnecTV Opus (PAL) [TeamEurope, David Haywood]
GameU Joint TV Bank 108-in-1 (model AH9069) [TeamEurope, David Haywood]

new NOT WORKING machines
----------------
Zuba Zuba Blade (Japan) [TeamEurope, David Haywood]
Beyblade Ultimate Shooter (Japan) [TeamEurope, David Haywood]
TV Hockey (Japan) [TeamEurope, David Haywood]
Let's Play Excite Striker (UK) [TeamEurope, David Haywood]
Play Portable Color GameU+ (108-in-1) (Japan) [TeamEurope, David Haywood]
Handheld Game Console 319-in-1 [TeamEurope, David Haywood]
Classic Max Pocket Mx-10 - 12 in 1 (vertical)  [TeamEurope, David Haywood]
XaviX Aerostep (XaviXPORT, Japan) [TeamEurope, David Haywood]
Tamagotchi Pix [TeamEurope, David Haywood]
BornKid 16 Bit Handheld Games 100-in-1 [TeamEurope, David Haywood]
Sumikko Gurashi - Sumikko Sagashi [TeamEurope, David Haywood]

new software list additions
----------------------
ekara_cart.xml:
e-kara Web cartridge 12M (used, with 5 songs) (Japan)  [TeamEurope, David Haywood]

### CODE REVIEW

**Comment by mamehaze** [comment]
tak_beyb is
ベイブレード　アルティメットシューター

tak_zuba is
ズバズバブレード

tom_jump is
テレビで遊び隊　韋駄天翔 激走 韋駄天バトル

**Comment by mamehaze** [comment]
TV Hockey is kinda playable, but the controls still seem a bit off (beyond MAME's usual poor handling of Analog inputs)

**Comment by mamehaze** [comment]
cart+pcb
the 5 songs on this user cart

**Comment by mamehaze** [comment]
I need a proper title for this thing
https://omokore.shop/2019/05/27/%E3%80%8C%E3%81%99%E3%81%BF%E3%81%A3%E3%82%B3%E3%81%95%E3%81%8C%E3%81%97%E3%80%8D%E3%82%AB%E3%83%A1%E3%83%A9%E3%81%A7%E3%81%99%E3%81%BF%E3%81%A3%E3%82%B3%E7%99%BA%E8%A6%8B%EF%BC%81%E3%81%99%E3%81%BF/

**Comment by einstein95** [comment]
> I need a proper title for this thing https://omokore.shop/2019/05/27/%E3%80%8C%E3%81%99%E3%81%BF%E3%81%A3%E3%82%B3%E3%81%95%E3%81%8C%E3%81%97%E3%80%8D%E3%82%AB%E3%83%A1%E3%83%A9%E3%81%A7%E3%81%99%E3%81%BF%E3%81%A3%E3%82%B3%E7%99%BA%E8%A6%8B%EF%BC%81%E3%81%99%E3%81%BF/

Appears to be called "Sumikko Gurashi - Sumikko Surashi" (すみっコぐらし すみっコさがし) [JAN: 4904810499220], released in 2019
http://web.archive.org/web/20200808091730/https://www.takaratomy.co.jp/products/sumikko/goods/game.html

**Comment by mamehaze** [comment]
@cuavas can you verify / validate the Japanese titles listed above, and add the tracklist for the e-kara user cart?

File: src/mame/tvgames/xavix.cpp [hunk 1]
```diff
@@ -2526,15 +2602,20 @@ CONS( 2006, rad_sbw,   0,          0,  xavix,            rad_snow, xavix_state,
...
-// A UK branded Epoch version (no Radica involvement) also exists, but so far all units seen have been 'no TSOP pads' cases
+CONS( 2002, epo_strkp, epo_strk,   0,  xavixp,           epo_epp,  xavix_state,          init_xavix,    "Epoch / SSD Company LTD",                      "Let's Play Excite Striker (UK)", MACHINE_NOT_WORKING | MACHINE_IMPERFECT_SOUND )
+CONS( 2002, rad_socr,  epo_strk,   0,  xavix,            epo_epp,  xavix_state,          init_xavix,    "Radica / Epoch / SSD Company LTD",             "Play TV Soccer", MACHINE_NOT_WORKING | MACHINE_IMPERFECT_SOUND )
 
-CONS( 2002, rad_jcon,  0,          0,  xavix,            rad_jcon, xavix_state,          init_xavix,    "Radica / Takara / SSD Company LTD",            "Play TV Jr. Construction", MACHINE_IMPERFECT_SOUND )
+// まちをつくろう 工事のくるま５     
+CONS( 2001, tak_town,  0,          0,  xavix,            rad_jcon, xavix_state,          init_xavix,    "Takara / SSD Company LTD",                     "Let's Build a Town: Construction Cars 5 (Japan)", MACHINE_IMPERFECT_SOUND )
```
Comment (cuavas) [inline src/mame/tvgames/xavix.cpp:2611]: “Machi o Tsukurou: Kouji no Kuruma 5”

**Comment by cuavas** [comment]
> @cuavas can you verify / validate the Japanese titles listed above, and add the tracklist for the e-kara user cart?

Just two Japanese game titles need attention – see comments.  I’ll add the track list for the user cartridge tomorrow.

File: src/mame/tvgames/xavix_2000.cpp [hunk 2]
```diff
@@ -390,6 +396,10 @@ CONS( 2005, ttv_sw,   0,      0, xavix2000_i2c_24c02, ttv_lotr,    xavix_i2c_lot
 CONS( 2005, ttv_swj,  ttv_sw, 0, xavix2000_i2c_24c02, ttv_lotr,    xavix_i2c_lotr_state,    init_xavix, "Tomy / SSD Company LTD",        "Star Wars Saga Edition - Lightsaber Battle Game (Japan)", MACHINE_NOT_WORKING | MACHINE_IMPERFECT_SOUND )
 CONS( 2005, ttv_lotr, 0,      0, xavix2000_i2c_24c02, ttv_lotr,    xavix_i2c_lotr_state,    init_xavix, "Tiger / SSD Company LTD",       "Lord Of The Rings - Warrior of Middle-Earth", MACHINE_NOT_WORKING | MACHINE_IMPERFECT_SOUND )
 CONS( 2005, ttv_mx,   0,      0, xavix2000_i2c_24c04, ttv_mx,      xavix_i2c_state,         init_xavix, "Tiger / SSD Company LTD",       "MX Dirt Rebel", MACHINE_IMPERFECT_SOUND )
+
+// テレビで遊び隊　韋駄天翔 激走 韋駄天バトル  - seems to be based on the same engine at ttv_mx and has an almost identical controller, but not exactly the same game
+CONS( 2005, tom_jump,   0,    0, xavix2000_i2c_24c04, ttv_mx,      xavix_i2c_state,         init_xavix, "Tomy / SSD Company LTD",        "Idaten Sho Gekisou - Idaten Jump (Japan)", MACHINE_IMPERFECT_SOUND )
```
[the remark this answers is not in the document]
Comment (mamehaze) [reply src/mame/tvgames/xavix_2000.cpp:401]: the Japanese names we have came from the old SSD master list on their website (which is now offline)

[the remark this answers is not in the document]
Comment (cuavas) [reply src/mame/tvgames/xavix_2000.cpp:401]: Yeah, the Japanese name in the comment looks sane, it just doesn’t match what you put in the description.

[the remark this answers is not in the document]
Comment (mamehaze) [reply src/mame/tvgames/xavix_2000.cpp:401]: this is the titlescreen

[the remark this answers is not in the document]
Comment (mamehaze) [reply src/mame/tvgames/xavix_2000.cpp:401]: https://muuseo.com/henly/items/503  is the physical product

**Pull request merged**